contracts/staking/stakeManager/StakeManager.sol

```diff
@@ -304,12 +304,12 @@ contract StakeManager is IStakeManager {
         );
     }
 
+    /// @notice currently replace with just minimum amount
```

Comment:
Not clear from the comment what the function was intended for. I'd suggest simply removing the function for now for clarity.

Reply:
this comment is not relevant since mechanism/code is updated.